pm/sendermonitor.go

```diff
@@ -216,6 +230,28 @@ func (sm *LocalSenderMonitor) reserveAlloc(addr ethcommon.Address) (*big.Int, er
 	return new(big.Int).Sub(new(big.Int).Div(reserve, poolSize), claimed), nil
 }
 
+// Returns the current available funds for a sender that could cover redemptions
+func (sm *LocalSenderMonitor) availFunds(addr ethcommon.Address) (*big.Int, error) {
```

Comment:
nit: Can we just name this `availableFunds`. Four letters don't seem to matter that much in terms of length.

Reply:
Fixed 6e568b4